src/manifold/src/constructors.cpp

```diff
@@ -92,7 +94,9 @@ Manifold Manifold::Smooth(const MeshGL64& meshGL64,
                "rather than Smooth().");
 
   std::shared_ptr<Impl> impl = std::make_shared<Impl>(meshGL64);
-  impl->CreateTangents(impl->UpdateSharpenedEdges(sharpenedEdges));
+  const Vec<bool> internalEdges = impl->InternalEdges();
+  impl->CreateTangents(impl->UpdateSharpenedEdges(sharpenedEdges),
+                       internalEdges);
```

Comment:
@pca006132 Any ideas for reducing the duplication between these two `MeshGL` functions?

Reply:
Move this to impl?